create-hpc-cluster-linux-cn/azuredeploy.json

```diff
@@ -0,0 +1,455 @@
+                "CentOS_6.6",
+                "CentOS_7.0",
+                "SLES_12",
+                "SLES_Premium_12",
+                "SLES_HPC_12",
+                "SLES_HPC_Premium_12",
+                "Ubuntu_14.04"
+            ],
+            "metadata": {
+                "description": "The Linux distro of the compute nodes"
+            }
+        },
+        "computeNodeNamePrefix": {
+            "type": "string",
+            "defaultValue": "IaaSLnxCN-",
+            "metadata": {
+                "description": "The name prefix of the compute nodes. It can contain letters, numbers and hyphens and must start with a letter, up to 13 characters. For example, if 'IaaSLnxCN-' is specified, the compute node names will be 'IaaSLnxCN-0', 'IaaSLnxCN-1', ..."
+            }
+        },
+        "computeNodeNumber": {
+            "type": "int",
+            "defaultValue": 2,
+            "metadata": {
+                "description": "The number of the compute nodes"
+            }
+        },
+        "computeNodeVMSize": {
+            "type": "string",
+            "defaultValue": "Standard_A3",
+            "allowedValues": [
+                "Standard_A1",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4",
+                "Standard_A5",
+                "Standard_A6",
+                "Standard_A7",
+                "Standard_A8",
+                "Standard_A9",
+                "Standard_A10",
+                "Standard_A11",
+                "Standard_D1",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4",
+                "Standard_D11",
+                "Standard_D12",
+                "Standard_D13",
+                "Standard_D14"
+            ],
+            "metadata": {
+                "description": "The VM role size of the compute nodes"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "defaultValue": "azureuser",
+            "metadata": {
+                "description": "The user name of the administrator"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "defaultValue": "Pa$$word123",
+            "metadata": {
+                "description": "The password of the administrator"
+            }
+        },
+        "headNodePostConfigScript": {
+            "type": "string",
+            "defaultValue": " ",
+            "metadata": {
+                "description": "Optional, specify the script url and command line if you want to run your custom script on the head node after it is configured. The script url must be public available, and you can also specify arguments for the script, for example 'http://www.consto.com/mypostscript.ps1 -Arg1 arg1 -Arg2 arg2'."
+            }
+        }
+    },
+
+    "variables": {
+        "storageAccountName": "[concat(toLower(parameters('clusterName')), 'sa')]",
```

Comment:
This should be a parameter as it needs to be unique. User could run into deployment failure in case they don't realize that the clusterName needs to be unique.